packages/cadl-ranch-specs/http/azure/core/lro/rpc/main.tsp

```diff
@@ -132,90 +134,10 @@ Expected response body:
 """)
 @route("/same-poll-result")
 interface SamePollResult {
-  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Will use Azure.Core when LongRunningRpcOperation is ready"
-  @doc("Creates a Job")
-  @route("/jobs")
-  @global.Azure.Core.pollingOperation(SamePollResult.getJob)
-  createJob is LocalCore.LongRunningRpcOperation<JobData, JobResult>;
-
-  @doc("Gets the status of a Job")
-  getJob is ResourceOperations.ResourceRead<JobResult>;
-}
-
-@scenario
-@doc("Creates a Job")
-@summary("Creates a Job")
-@scenarioDoc("""
-Expected verb: POST
-Expected request body:
-```json
-{
-  "comment": "async job"
-}
-```
-
-Expected status code: 202
-Expected response header: operation-location={endpoint}/different-poll-result/jobs/operations/operation1
-Expected response header: location={endpoint}/different-poll-result/jobs/job1
-Expected response body:
-```json
-{
-  "operationId": "operation1",
-  "status": "InProgress"
-}
-```
-
-Expected verb: GET
-Expected URL: {endpoint}/different-poll-result/jobs/operations/operation1
-
-Expected status code: 200
-Expected response body:
-```json
-{
-  "operationId": "operation1",
-  "status": "InProgress"
-}
-```
-
-Expected verb: GET
-Expected URL: {endpoint}/different-poll-result/jobs/operations/operation1
-
-Expected status code: 200
-Expected response body:
-```json
-{
-  "operationId": "operation1",
-  "status": "Succeeded"
-}
-```
-
-Expected verb: GET
-Expected URL: {endpoint}/different-poll-result/jobs/job1
-
-Expected status code: 200
-Expected response body:
-```json
-{
-  "jobId": "job1",
-  "comment": "async job",
-  "status": "Succeeded",
-  "results": ["job1 result"]
-}
-```
-""")
-@route("/different-poll-result")
-interface DifferentPollResult {
-  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Will use Azure.Core when LongRunningRpcOperation is ready"
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Legacy LRO uses local template"
```

Comment:
The above doc may need to change. the status should be changed from `inprogress` to `running`

Reply:
Thanks for catching this. Indeed I forgot update the scenarioDoc after mockapi change.